polaris/utils/v2_manifest.py

```diff
@@ -0,0 +1,71 @@
+from hashlib import md5
```

Comment:
What do you think about renaming this file to `manifest` and moving it to `polaris/dataset/zarr`? `utils` seems a little out of place but not terrible.

Reply:
Like it!